requirements.txt

```diff
@@ -5,6 +5,7 @@ cuquantum>=22.3.0
 flaky
 pennylane>=0.28
 pennylane-lightning>=0.28
+mpi4py
```

Comment:
I wonder if this should be made optionally in a seperate req file, as it may interfere with RTD builds

Reply:
Removed here.